<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gest_vehi</class>
 <widget class="QMainWindow" name="gest_vehi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>951</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string>Gesttion Des Vehicules</string>
    </property>
    <widget class="QPushButton" name="btn_supprimer">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>520</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>SUPPRIMER</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_mat_supp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>470</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>470</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_mat">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_marque">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Marque : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nom">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> Nom : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nb_porte">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre de portes : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat : </string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bonne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En Panne</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Couleur : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_puissance">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Puissance : </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_modifier">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>MODIFIER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ajouter">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>AJOUTER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri_id">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TRI ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_rechercher_1">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>460</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>460</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(C:/Users/21622/Desktop/pdficon.png);</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stat">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>460</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(C:/Users/21622/Desktop/staticon.png);</string>
     </property>
     <property name="text">
      <string>STAT</string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_voiture">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>661</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_couleur">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>460</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>490</y>
       <width>56</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Marque : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_rechercher_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>490</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>520</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre de portes : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_rechercher_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>520</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>550</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat : </string>
     </property>
    </widget>
    <widget class="QComboBox" name="le_rechercher_4">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>550</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bonne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En Panne</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>430</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
